client/.eslintrc.json

```diff
@@ -14,6 +14,6 @@
     "rules": {
         "react/react-in-jsx-scope": "off",
         "react/jsx-uses-react": "off",
-        "prettier/prettier": "error"
+        "prettier/prettier": 0
```

Comment:
We want to make sure we do not disable these checks when pushing our code. These are used as a styling check during the PR. If there are some linter or styling errors, you can always run the necessary commands to check and sometimes fix the issue. If you have any questions, let Daniel or myself know.